src/Linter/ProcessLintingResult.php

```diff
@@ -23,20 +23,14 @@
  */
 final class ProcessLintingResult implements LintingResultInterface
 {
-    /**
-     * @var bool
-     */
-    private $isSuccessful;
+    private Process $process;
 
-    /**
-     * @var Process
-     */
-    private $process;
+    private ?string $path;
 
     /**
-     * @var null|string
+     * @var bool
      */
-    private $path;
+    private $isSuccessful;
```

Comment:
```suggestion
    private bool $isSuccessful;
```

Reply:
thanks added, but with `= null`, see other comment for details